Makefile

```diff
@@ -1,40 +1,71 @@
-DESTDIR :=
+DEST_DIR :=
```

Comment:
Ironic as they were trying to be more legible, but ok I'll change these to the common values.